### PR TITLE
This solves issue 463

### LINKED ISSUE #463: Remove volume from db service in docker-compose, update readme.md
### What's your idea?

Currently a volume is configured in docker-compose to persist all the data from the postgres database container to a "postgres-data" folder in the project folder. 
Since this folder does not belong to the user in the docker context, whenever you run any docker commands it will cause errors. 
As we don't need to persist the postgres data in the dev environment (docker), we may remove the volume configuration from docker-compose.
Also, since it appears that using docker for the development environment is easier and more efficient, it is convenient to put the instructions on how to run the full project in docker at the beginning of the web repo readme.md. 
Those who prefer to run the project locally and install all the programs and dependencies in their computer, this can still be accomplished by following the instructions on readme.md. 
For those who get the below error when running the project "System has not been booted with systemd as init system (PID 1). Can't operate ", use `service <service.name> start` instead of `systemctl start <service.name>`

### Impact

High

### Urgency

Now

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by giulio-giunta**
This changes the docker-compose file by removing the volume from postgres db service and adding one in the web service instead. 
Also the readme.md has been modified to show the configuration to run the project in docker as a first option. 

"resolves #463 -->

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by giulio-giunta** [comment]
I needed to change executor in circleci from docker container to vm otherwise no volumes can be attached to the web service container in docker-compose

**Comment by jamesgeddes** [comment]
Remember to give PRs and issues a descriptive summary title rather than just a number. 😄

**Comment by giulio-giunta** [comment]
Ok. I was more descriptive in the issue, but not in the PR 😅

**Review by jamesgeddes — CHANGES_REQUESTED** [review]

File: readme.md [hunk 1]
```diff
@@ -8,9 +8,52 @@ Here's what the front page looks like in [light mode](/screencapture-gzweb-light
...
+```sh
+# Install command completion
+sudo curl \
+    -L https://raw.githubusercontent.com/docker/compose/1.29.2/contrib/completion/bash/docker-compose \
+    -o /etc/bash_completion.d/docker-compose
+source ~/.bashrc
+```
+##### Windows 10
+
+Follow the instructions in the Docker documentation https://docs.docker.com/desktop/windows/install/; the installation varies depending on the Windows 10 edition. Docker Desktop for Windows includes Compose, so there is no need to install it separately. Once the installion is complete, right-click on the Docker icon in the system tray, and `Switch to Linux Containers` if Docker Desktop is set to Windows Containers.
```
[the remark this answers is not in the document]
Comment (giulio-giunta) [reply readme.md:44]: Ok, I'll do that. The Docker documentation is already hyperlinked.

File: readme.md [hunk 2]
```diff
@@ -8,9 +8,52 @@ Here's what the front page looks like in [light mode](/screencapture-gzweb-light
 [dark mode](/screencapture-gzweb-dark-2021-03-22-20_24_03.png).
 
 ## Running the project locally
+The easiest and fastest way to run the project without cluttering your machine is by using docker containers. However you should be able to setup this project on any operating system that supports Django. We have instructions for Ubuntu based linux distributions and for Windows 10. Both can be found below.
 
-You should be able to setup this project on any operating system that supports Django. We have instructions for Ubuntu based linux distributions and for Windows 10. Both can be found below.
-Alternatively you can run this project in containers, by using docker-compose.
+#### Install docker & docker-compose
+
```
[the remark this answers is not in the document]
Comment (giulio-giunta) [reply readme.md:14]: done!

File: readme.md [hunk 3]
```diff
@@ -8,9 +8,52 @@ Here's what the front page looks like in [light mode](/screencapture-gzweb-light
...
+sudo curl \
+    -L https://raw.githubusercontent.com/docker/compose/1.29.2/contrib/completion/bash/docker-compose \
+    -o /etc/bash_completion.d/docker-compose
+source ~/.bashrc
+```
+##### Windows 10
+
+Follow the instructions in the Docker documentation https://docs.docker.com/desktop/windows/install/; the installation varies depending on the Windows 10 edition. Docker Desktop for Windows includes Compose, so there is no need to install it separately. Once the installion is complete, right-click on the Docker icon in the system tray, and `Switch to Linux Containers` if Docker Desktop is set to Windows Containers.
+
+### Running the project in docker containers
```
[the remark this answers is not in the document]
Comment (giulio-giunta) [reply readme.md:46]: Done!

File: readme.md [hunk 4]
```diff
@@ -8,9 +8,53 @@ Here's what the front page looks like in [light mode](/screencapture-gzweb-light
...
+```sh
+# Install command completion
+sudo curl \
+    -L https://raw.githubusercontent.com/docker/compose/1.29.2/contrib/completion/bash/docker-compose \
+    -o /etc/bash_completion.d/docker-compose
+source ~/.bashrc
+```
+##### Windows 10
+
+1. Follow the instructions in the Docker documentation https://docs.docker.com/desktop/windows/install/; the installation varies depending on the Windows 10 edition.
```
Comment (jamesgeddes) [inline readme.md:44]: The "Docker documentation" bit should be 

`[Docker documentation](https://docs.docker.com/desktop/windows/install)`

The use of 

`Docker documentation https://docs.docker.com/desktop/windows/install/`

means that we provide a raw URL.

**Review by jamesgeddes — CHANGES_REQUESTED** [review]

**Comment by giulio-giunta** [comment]
Ok, I see

File: readme.md (hunk 4 above)
Comment (giulio-giunta) [reply readme.md]: Done!

**Comment by giulio-giunta** [comment]
@jamesgeddes I made all the changes your suggested

**Review by jamesgeddes — APPROVED** [review]

**Comment by jamesgeddes** [comment]
@giulio-giunta Please rebase and merge :)

**Pull request merged**